app/pages/task-list/existingorder/ui.test.js

```diff
@@ -120,107 +130,358 @@ test('should render the order description details', async (t) => {
+  await t.navigateTo(pageUrl);
+
+  const secondTaskFirstItem = Selector('li[data-test-id="task-1-item-0"]');
+
+  await t
     .expect(secondTaskFirstItem.find('a').exists).ok()
     .click(secondTaskFirstItem.find('a'))
     .expect(getLocation()).eql(`http://localhost:1234${baseUrl}/organisation/order-id/ordering-party`);
 });
 
-test('should render the second task as "Organisation information" and "supplier" as link not text', async (t) => {
+test('should not render the second task and first item as complete when returned as incomplete from the API', async (t) => {
+  const mockOrderSummary = generateMockOrderSummary([{ id: 'ordering-party', status: 'incomplete' }]);
+  await pageSetup(t, true, mockOrderSummary);
+  await t.navigateTo(pageUrl);
+
+  const secondTaskFirstItemCompleteTag = Selector('[data-test-id="task-1-item-0-complete-tag"]');
+
+  await t
+    .expect(secondTaskFirstItemCompleteTag.exists).notOk();
+});
+
+test('should only render the second task and first item as complete when returned as complete from the API', async (t) => {
+  const mockOrderSummary = generateMockOrderSummary([{ id: 'ordering-party', status: 'complete' }]);
+  await pageSetup(t, true, mockOrderSummary);
+  await t.navigateTo(pageUrl);
+
+  const secondTaskFirstItemCompleteTag = Selector('[data-test-id="task-1-item-0-complete-tag"]');
+
+  await t
+    .expect(secondTaskFirstItemCompleteTag.exists).ok();
+});
+
+// Second Task Second Item Tests
+test('should render the second task and second item', async (t) => {
   await pageSetup(t, true);
   await t.navigateTo(pageUrl);
 
-  const taskList = Selector('[data-test-id="task-list"]');
   const secondTask = Selector('li[data-test-id="task-1"]');
   const secondTaskSecondItem = Selector('li[data-test-id="task-1-item-1"]');
 
   await t
-    .expect(taskList.exists).ok()
     .expect(secondTask.exists).ok()
     .expect(await extractInnerText(secondTask.find('h2 span'))).eql('2.')
     .expect(await extractInnerText(secondTask.find('h2 div'))).eql('Organisation information')
     .expect(secondTaskSecondItem.exists).ok()
-    .expect(await extractInnerText(secondTaskSecondItem)).eql('Provide Supplier information')
+    .expect(await extractInnerText(secondTaskSecondItem.find('span'))).eql('Provide Supplier information');
+});
+
+test('should always render the second task and second item as a link', async (t) => {
+  await pageSetup(t, true);
+  await t.navigateTo(pageUrl);
+
+  const secondTaskSecondItem = Selector('li[data-test-id="task-1-item-1"]');
+
+  await t
     .expect(secondTaskSecondItem.find('a').exists).ok()
     .click(secondTaskSecondItem.find('a'))
     .expect(getLocation()).eql(`http://localhost:1234${baseUrl}/organisation/order-id/supplier`);
 });
 
-test('should render the third task as "Commencement date" and "data" as text not link', async (t) => {
+test('should not render the second task and second item as complete when returned as incomplete from the API', async (t) => {
```

Comment:
should not render the complete tag for ... when ...

Reply:
I think it would simplify the test descriptions if we went with task 1 item 1 instead of the words `first` `second` etc

Reply:
yeah that's a good shout 👍